src/v1_0/rw/ctrl.cpp

```diff
@@ -12,4 +12,20 @@ namespace abb :: rws :: v1_0 :: rw :: ctrl
 
         client.httpPost(uri.str(), content.str(), {Poco::Net::HTTPResponse::HTTP_OK});
     }
-}
\ No newline at end of file
+
+    SafetyViolationInfo getSafetyViolationInfo(RWSClient& client)
+    {
+        std::stringstream uri;
+        uri << Services::CTRL << "/safety/violation";
+
+        RWSResult rws_result = parseXml(client.httpGet(uri.str()).content());
+        
+        SafetyViolationInfo result;
+        result.unsynchronized = std::stoi(xmlFindTextContent(rws_result, XMLAttribute(Identifiers::CLASS, "unsynchronized"))) == 1;
```

Comment:
Maybe for it to be more according to how C++ handles bools it would make sense to parse it as:
```suggestion
        result.unsynchronized = std::stoi(xmlFindTextContent(rws_result, XMLAttribute(Identifiers::CLASS, "unsynchronized"))) != 0;
```